visualization-design.typ

```diff
@@ -0,0 +1,36 @@
+= Visualization Design
+
+The visualization serves a dual purpose: To communicate the functionality of the load balancer at a glance to a non-technical audience, and to serve as a useful debugging tool during development.
+
+== Hosting
+
+Option 1: Self-Hosted
+- Pros: No added cost, no compromises required   
+- Cons: Not always online, longer startup time
+
+Option 2: Buy a domain
+- Pros: Always online, easy access
+- Cons: Cost (that's probably enough of a con)
```

Comment:
Buying a domain name does not provide hosting. It's also not necessary, we can create infinite subdomains of `opentogethertube.com`, which I already pay for.